test/functional/api/version_spec.lua

```diff
@@ -119,60 +116,76 @@ describe("api metadata", function()
     end
   end)
 
-  it("functions are compatible with old metadata or have new level", function()
+  it('functions are compatible with old metadata or have new level', function()
     local funcs_new = name_table(api.functions)
     local funcs_compat = {}
     for level = compat, stable do
-      for _,f in ipairs(old_api[level].functions) do
+      for _, f in ipairs(old_api[level].functions) do
         if funcs_new[f.name] == nil then
           if f.since >= compat then
-            error('function '..f.name..' was removed but exists in level '..
-                  f.since..' which nvim should be compatible with')
+            error(
+              'function '
+                .. f.name
+                .. ' was removed but exists in level '
+                .. f.since
+                .. ' which nvim should be compatible with'
+            )
```

Comment:
This is a bit unfortunate (and follows no comprehensible rule). Could it be changed to not do that?

Reply:
No.

Reply:
(except by increasing the admissible line length)

Reply:
I'd have to check, but I assume it follows the same rule as for argument lists (`foo(a,b,c,d)`): if the full list in parentheses does not fit into the same line, every item gets its own line.

Reply:
https://github.com/JohnnyMorganz/StyLua/issues/139